openedx/core/djangoapps/team_api/models.py

```diff
@@ -0,0 +1,88 @@
+"""
+Django models related to teams functionality.
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from .utils import slugify
+
+from xmodule_django.models import CourseKeyField
+
+
+class CourseTeam(models.Model):
+    """
+    This model represents team related info.
+    """
+
+    class Meta:
+        unique_together = (('team_id', 'course_id'),)
+
+    team_id = models.CharField(max_length=100, db_index=True)
+    name = models.CharField(max_length=100)
+    is_active = models.BooleanField(default=True)
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    topic_id = models.CharField(max_length=100, db_index=True, blank=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    # last_activity is computed through a query
+    description = models.CharField(max_length=1000)
```

Comment:
Did you consider TextField instead of CharField? Honestly I'm not sure which is better, but it is worth investigating.